src/libraries/System.Drawing.Common/tests/ImageTests.cs

```diff
@@ -157,6 +157,24 @@ public static IEnumerable<object[]> GetEncoderParameterList_ReturnsExpected_Test
                     new Guid(unchecked((int)0xa219bbc9), unchecked((short)0x0a9d), unchecked((short)0x4005), new byte[] { 0xa3, 0xee, 0x3a, 0x42, 0x1b, 0x8b, 0xb0, 0x6c }) /* Encoder.SaveAsCmyk.Guid */
                 }
             };
+
+            if (!PlatformDetection.IsNetFramework)
+            {
+                // NetFX doesn't support pointer-type encoder parameters, and doesn't define Encoder.ImageItems. Skip this test
+                // on NetFX.
+                yield return new object[]
+                {
+                    ImageFormat.Jpeg,
+                    new Guid[]
+                    {
+                        Encoder.Transformation.Guid,
+                        Encoder.Quality.Guid,
+                        Encoder.LuminanceTable.Guid,
+                        Encoder.ChrominanceTable.Guid,
+                        Encoder.ImageItems.Guid
```

Comment:
So, the `ImageItems` property was added in this PR, but is not present in the `NetFramework` builds since they type forward to System.Drawing, which does not have this property.

How should we tackle this?